e2e-tests/tfm/cypress/e2e/journeys/case-amendments/show-amendment-button/deal-cancellation-enabled/show-amendment-button.spec.js

```diff
@@ -0,0 +1,44 @@
+import relative from '../../../../relativeURL';
+import facilityPage from '../../../../pages/facilityPage';
+import amendmentsPage from '../../../../pages/amendments/amendmentsPage';
+import MOCK_DEAL_AIN from '../../../../../fixtures/deal-AIN';
+import { PIM_USER_1, BANK1_MAKER1, ADMIN } from '../../../../../../../e2e-fixtures';
+import { submitDealCancellation } from '../../../../../support/trade-finance-manager-ui/submit-deal-cancellation';
+
+context('Amendments page', () => {
+  describe('A cancelled deal', () => {
+    let dealId;
+    const dealFacilities = [];
+
+    before(() => {
+      cy.insertOneDeal(MOCK_DEAL_AIN, BANK1_MAKER1).then((insertedDeal) => {
+        dealId = insertedDeal._id;
+
+        const { dealType, mockFacilities } = MOCK_DEAL_AIN;
+
+        cy.createFacilities(dealId, mockFacilities, BANK1_MAKER1).then((createdFacilities) => {
+          dealFacilities.push(...createdFacilities);
+        });
+
+        cy.submitDeal(dealId, dealType, PIM_USER_1);
+        submitDealCancellation({ dealId });
+      });
+    });
+
+    after(() => {
+      cy.deleteDeals(dealId, ADMIN);
```

Comment:
So many questions:
- why is this called deleteDeals when it only takes one
- is this deleting the TFM deal too?
- is it deleting the facilities? I assume so given it calls an api
- Do we even want to delete the deal? Makes it harder to debug after the tests are run locally